utah-design-system-website/src/react/components/websiteContent/Search.jsx

```diff
@@ -19,9 +20,56 @@ export function Search() {
     if (param) { setQuery(param); }
   };
 
+  const checkSkipLink = () => {
+    const link = document.getElementById('skip-link');
+    if (link) {
+      setSkipLink(link);
+    } else {
+      // Create observer to watch for new nodes
+      const obs = new MutationObserver(() => {
+        const link = document.getElementById('skip-link');
+        if (link) {
+          setSkipLink(link);
+        }
+      });
+      // @ts-ignore
+      if (!observer) setObserver(obs);
+    }
+  };
+
+  const setSkipLink = (/** @type {HTMLElement} */ link) => {
+    // Update skip link
+    link.onclick = (e) => {
+      e.preventDefault();
+      // Focus on search input
+      const content = document.getElementById('searchInput');
+      content?.focus();
+    };
+    // Disconnect observer when done
+    // @ts-ignore
+    if (observer) observer.disconnect();
+  };
+
+  useEffect(() => {
+    if (observer) {
+      // @ts-ignore
+      observer.observe(document, { subtree: true, childList: true });
+    }
+  }, [observer]);
+
   useEffect(() => {
     insertScript();
     getSearchTerm();
+    checkSkipLink();
+  }, []);
+
+  useEffect(() => () => {
+    // Disconnect observer on unmount
+    // @ts-ignore
+    if (observer) observer.disconnect();
+    // Reset link behavior
+    const link = document.getElementById('skip-link');
+    if (link) link.onclick = () => null;
   }, []);
 
   return (
```

Comment:
on line 85, the search button is hidden if there isn't any search text. hidden controls can sometimes be awkward for users trying to find functionality? what if the button was disabled instead of hidden until a search was entered? or maybe even better, if it shows a message when they click it if there isn't text entered, that they need to enter text. what are our form validation guidelines for something like this?